Packages/com.unity.inputsystem/InputSystem/Devices/Gamepad.cs

```diff
@@ -402,106 +399,146 @@ namespace UnityEngine.InputSystem
     ///
     /// <example>
     /// <code>
-    /// // Show all gamepads in the system.
-    /// Debug.Log(string.Join("\n", Gamepad.all));
     ///
-    /// // Check whether the X button on the current gamepad is pressed.
-    /// if (Gamepad.current.xButton.wasPressedThisFrame)
-    ///     Debug.Log("Pressed");
+    /// using UnityEngine;
+    /// using UnityEngine.InputSystem;
     ///
-    /// // Rumble the left motor on the current gamepad slightly.
-    /// Gamepad.current.SetMotorSpeeds(0.2f, 0.
+    /// public class Example : MonoBehaviour
+    /// {
+    ///     void Start()
+    ///     {
+    ///         // Print all connected gamepads
+    ///         Debug.Log(string.Join("\n", Gamepad.all));
+    ///     }
+    ///
+    ///     void Update()
+    ///     {
+    ///         var gamepad = Gamepad.current;
+    ///
+    ///         // No gamepad connected.
+    ///         if (gamepad == null)
+    ///         {
+    ///             return;
+    ///         }
+    ///
+    ///         // Check if "Button North" was pressed this frame
+    ///         if (gamepad.buttonNorth.wasPressedThisFrame)
+    ///         {
+    ///             Debug.Log("Button North was pressed");
+    ///         }
+    ///
+    ///         // Check if the button control is being continuously actuated and read its value
+    ///         if (gamepad.rightTrigger.IsActuated())
+    ///         {
+    ///             Debug.Log("Right trigger value: " + gamepad.rightTrigger.ReadValue());
+    ///         }
+    ///
+    ///         // Read left stick value and perform some code based on the value
+    ///         Vector2 move = gamepad.leftStick.ReadValue();
+    ///         {
```

Comment:
why { } if there is no control statement?

Reply:
It's just to scope out  the`move` variable, as seen in an example [here](https://docs.unity3d.com/Packages/com.unity.inputsystem@1.11/manual/Workflow-Direct.html)